Borrowing-history.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Borrowing History</title>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+<script>
+  function myFunction() {
+   var x = document.getElementById("myTopnav");
+      if (x.className === "topnav") {
+            x.className += " responsive";
+            } else {
+                x.className = "topnav";
+              }
+        }
+</script>
+<style>
+    body {font-family: Helvetica, sans-serif;}
```

Comment:
I would love to see some DRY code. check this out https://code.tutsplus.com/tutorials/3-key-software-principles-you-must-understand--net-25161.